app/src/main/res/values/donottranslate.xml

```diff
@@ -86,6 +86,9 @@
     <!-- Description for Arabic (PC) subtype. -->
     <string name="subtype_arabic_pc" translatable="false">%s (PC)</string>
 
+    <!-- Description for Arabic (Hija'i) subtype. -->
+    <string name="subtype_arabic_hijai" translatable="false">%s (Hija'i)</string>
```

Comment:
Please escape the `'`, otherwise when compiling you'll get _Failed to flatten XML for resource 'subtype_arabic_hijai' with error: Invalid unicode escape sequence in string_